redaxo/src/addons/users/pages/users.php

```diff
@@ -442,17 +442,17 @@
 
     $n = [];
     $n['label'] = '<label for="rex-user-name">' . rex_i18n::msg('name') . '</label>';
-    $n['field'] = '<input class="form-control" type="text" id="rex-user-name" name="username" value="' . rex_escape($username) . '" />';
+    $n['field'] = '<input class="form-control" type="text" id="rex-user-name" name="username" value="' . rex_escape($username) . '"  autocomplete="name" />';
     $formElements[] = $n;
 
     $n = [];
     $n['label'] = '<label for="rex-user-description">' . rex_i18n::msg('description') . '</label>';
-    $n['field'] = '<input class="form-control" type="text" id="rex-user-description" name="userdesc" value="' . rex_escape($userdesc) . '" />';
+    $n['field'] = '<input class="form-control" type="text" id="rex-user-description" name="userdesc" value="' . rex_escape($userdesc) . '" autocomplete="organization" />';
```

Comment:
Warum hier organization?

Reply:
Wenn es kein passendes Attribut gibt, bitte weglassen

Reply:
Ich wollte es damit zur Diskussion stellen. Es wäre ein sinnvoller Vorschlag, der über autocomplete hinzukommt. Kann's aber auch weglassen. Mach doch bei so etwas gleich eine inline-Korrektur
```suggestion
    $n['field'] = '<input class="form-control" type="text" id="rex-user-description" name="userdesc" value="' . rex_escape($userdesc) . '" autocomplete="off" />';
```

Reply:
Ganz ehrlich? Ich weiß garnicht wie eine inline Korrektur funktioniert.